src/ui/localization/supportedLanguages.ts

```diff
@@ -78,9 +78,10 @@ export enum LocalizationKeys {
   PaymentEntryPageButtonPaymentMethod = "payment_entry_page.button_payment_method",
   PaymentEntryPageExpressCheckoutDivider = "payment_entry_page.express_checkout_divider",
   SuccessPagePurchaseSuccessful = "success_page.purchase_successful",
-  SuccessPageSubscriptionNowActive = "success_page.subscription_now_active",
```

Comment:
We agreed to remove this text as the following screen provides the same information.